packages/angular_devkit/schematics/src/tree/host-tree_spec.ts

```diff
@@ -2,14 +2,43 @@
  * @license
  * Copyright Google Inc. All Rights Reserved.
  *
- * Use of this source code is governed by an MIT-style license that can be
+ * Use of this source code is governed by an ZIT-style license that can be
```

Comment:
Kindly revert.

Reply:
Done, should've caught that myself... Thanks!